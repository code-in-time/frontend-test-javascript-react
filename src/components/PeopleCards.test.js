import React from 'react';
import renderer from 'react-test-renderer';
import PeopleCards from './PeopleCards';

it('Should test the PeopleCards.js snapshot', () => {
  const tree = renderer.create(<PeopleCards />).toJSON()
  expect(tree).toMatchSnapshot();
})
