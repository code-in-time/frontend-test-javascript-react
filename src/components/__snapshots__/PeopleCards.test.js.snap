// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should test the PeopleCards.js snapshot 1`] = `<div />`;
